### PR TITLE
Add object model for Screening* entries

### PR DESCRIPTION
**Pull request by rjgildea**
Add object models for the following database tables:
- Screening
- ScreeningOutput
- ScreeningOutputLattices
- ScreeningStrategy
- ScreeningStrategyWedge
- ScreeningStrategySubWedge

### CODE REVIEW

File: ispyb/model/screening.py [hunk 1]
```diff
@@ -0,0 +1,394 @@
...
+        self._screening_id = int(screening_id)
+        if preload:
+            self._data = preload
+
+    def reload(self):
+        """Load/update information from the database."""
+        raise NotImplementedError()
+
+    @property
+    def screening_outputs(self):
```
Comment (Anthchirp) [inline ispyb/model/screening.py]: given that you have a `screenings` type object we can probably just call this property `.outputs()`

File: ispyb/model/screening.py [hunk 2]
```diff
@@ -0,0 +1,394 @@
...
+                )
+            )
+        ).format(self)
+
+
+ispyb.model.add_properties(
+    Screening,
+    (
+        ("comments", "comments"),
+        ("short_comments", "shortComments"),
```
Comment (Anthchirp) [inline ispyb/model/screening.py]: `comments` and `short_comments` are just strings. Assert `comment` and `short_comment` are clearer

File: tests/model/test_screening.py [hunk 3]
```diff
@@ -0,0 +1,195 @@
...
+def test_model_screening(testdb, testconfig):
+    ispyb.model.__future__.enable(testconfig)
+    dc = ispyb.model.datacollection.DataCollection(1052494, testdb)
+    assert len(dc.screenings) == 7
+
+    for screening in (
+        dc.screenings[2],
+        ispyb.model.screening.Screening(dc.screenings[2]._screening_id, testdb),
+    ):
+        assert screening.program_version == "EDNA MXv1"
```
Comment (Anthchirp) [inline tests/model/test_screening.py]: would `.program` be clearer than `.program_version`, given that the example string includes the program name?

File: tests/model/test_screening.py [hunk 4]
```diff
@@ -0,0 +1,195 @@
...
+    assert len(screening_output.strategies) == 1
+
+    for sol in (
+        screening_output.lattices[0],
+        ispyb.model.screening.ScreeningOutputLattice(
+            screening_output.lattices[0]._lattice_id, testdb
+        ),
+    ):
+        assert sol.unit_cell.a == 76.3
+        assert sol.unit_cell.a == 76.3
```
Comment (Anthchirp) [inline tests/model/test_screening.py]: redundant line

File: tests/model/test_screening.py [hunk 5]
```diff
@@ -0,0 +1,195 @@
...
+        assert sub_wedge.number_of_images == 220
+        assert sub_wedge.oscillation_range == 0.15
+        assert sub_wedge.resolution == 1.41
+        assert sub_wedge.sub_wedge_number == 1
+        assert sub_wedge.transmission == 100.0
+
+
+def test_model_screening_str(testdb, testconfig):
+    ispyb.model.__future__.enable(testconfig)
+    screening = ispyb.model.screening.Screening(1894774, testdb)
```
Comment (Anthchirp) [inline tests/model/test_screening.py]: we should have proper accessor functions added to https://github.com/DiamondLightSource/ispyb-api/blob/master/ispyb/model/interface.py

**Review by Anthchirp — APPROVED** [review]
looks alright to me

**Review by KarlLevik — APPROVED** [review]
:+1:

File: tests/model/test_screening.py [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+    ispyb.model.__future__.enable(testconfig)
+    screening = testdb.get_screening(1894774)
+    screening.load()
+    assert (
+        str(screening)
+        == """\
+Screening #1894774
+  comment         : Standard Native Dataset Multiplicity=3 I/sig=2 Maxlifespan=202 s
+  short_comment   : EDNAStrategy1
+  program  : EDNA MXv1"""
```
Comment (Anthchirp) [inline tests/model/test_screening.py]: can we realign these?

**Review by Anthchirp — APPROVED** [review]

**Pull request merged**